client/gutenberg/extensions/tiled-gallery/tiled-gallery.scss

```diff
@@ -0,0 +1,62 @@
+.tiled-gallery-square {
+	clear: both;
+	margin: 0 0 20px;
+	overflow: hidden;
+}
+img {
```

Comment:
even with the namespace prefix won't this bleed into the `img` elements from other blocks?

Reply:
consider pulling this into the `p2`/`o2` preset or the Jetpack preset - it would end up as `.o2 img { }`

Reply:
Yeah, I agree here - ideally we shouldn't be targeting all `img`s, but rather have a dedicated className for these images.